scio-jdbc/src/main/scala/com/spotify/scio/jdbc/package.scala

```diff
@@ -0,0 +1,153 @@
+package com.spotify.scio
+
+import java.sql.{Driver, PreparedStatement, ResultSet}
+
+import com.spotify.scio.Implicits._
+import com.spotify.scio.io.Tap
+import com.spotify.scio.testing.TestIO
+import com.spotify.scio.values.SCollection
+import org.apache.beam.sdk.io.jdbc.JdbcIO
+import org.apache.beam.sdk.io.jdbc.JdbcIO._
+
+import scala.concurrent.Future
+import scala.reflect.ClassTag
+
+package object jdbc {
+
+  val TEST_READ_TABLE_NAME = "table_read"
+  val TEST_WRITE_TABLE_NAME = "table_write"
+
+  /**
+    * Options require to create a connection with remote database.
```

Comment:
Use JavaDoc style please. Like rest of the code base. You can configure IntelliJ to do that for Scala code.
```scala
/**
 * Blah ...
 */
```